jib-cli/src/test/java/com/google/cloud/tools/jib/cli/buildfile/BuildFileSpecTest.java

```diff
@@ -93,7 +93,7 @@ public void testBuildFileSpec_full() throws JsonProcessingException {
   public void testBuildFileSpec_apiVersionRequired() {
     String data = "kind: BuildFile\n";
 
-    JsonProcessingException exception =
+    Exception exception =
```

Comment:
Can we leave these as `JsonProcessingException`?

Reply:
What's the reason for that? We are only checking the exception message. In some sense, it's similar to `List list = ...` instead of `ArrayList list = ...`.

Reply:
This might be more of a personal choice but sometimes it can be helpful to be specific about the exception being returned. Sometimes our method can return a different exception, causing the tests to pass even when it is not supposed to. However, in this case it might be okay.

Reply:
Oh, that's not a concern. We explicitly define the expected exception as the first argument for `assertThrows`. It's only that we just use the type `Exception` to hold the returned instance. The instance will always be `JsonProcessingException`.
```java
    Exception exception =
        assertThrows(
            JsonProcessingException.class, () -> mapper.readValue(data, BuildFileSpec.class));
```

Reply:
Ah yes, you're right. In that case, using a generic type shouldn't be problem.